src/routes/settings/assistants/[assistantId]/+page.server.ts

```diff
@@ -64,9 +64,10 @@ export const actions: Actions = {
 			return fail(400, { error: true, message: "Already reported" });
 		}
 
-		const reason = (await request.formData()).get("reportReason"); // get the reportReason and validate it
+		const formData = await request.formData();
+		const result = z.string().min(1).max(128).safeParse(formData?.get("reportReason"));
 
-		if (!reason || typeof reason !== "string" || reason.length > 128 || reason.length < 1) {
+		if (!result.success) {
```

Comment:
is there something like `result.message` that can be used in the line below rather than the generic "Invalid report reason"?

Reply:
closing it since in the UI, we already have `maxlength="128"`

Reply:
sorry I was taking a look, but we don't show error messages so I think the client side checks we have are currently enough (max length & disabled button on empty)